background.js

```diff
@@ -1,49 +1,82 @@
-browser.pageAction.onClicked.addListener(async () => {
-  await browser.tabs.executeScript({
-    code: `document.getElementsByClassName("download")[0].click();`,
-  });
+const showMessage = async message => {
+  await browser.tabs.executeScript({code: `
+    closeBtn = document.createElement("span");
+    closeBtn.textContent = "×";
+    closeBtn.setAttribute("style", "font-weight:bold;float:right;font-size:20px;line-height:18px;cursor:pointer;padding-left:15px;");
+    closeBtn.setAttribute("onclick", "this.parentElement.style.display='none';");
```

Comment:
This is certainly a non-standard way to attach a `click` handler :smile: The idiomatic way to do it would be:

```suggestion
    closeBtn.addEventListener('click', (event) => {
      alertBox.parentElement.removeChild(alertBox);
    });
```

Note that this also changes the way the element is removed - it is removed from the DOM rather than keeping the element hidden. Theoretically, if there would be hundreds of messages, they would stay in the DOM unnecessarily, increasing memory usage and slowing the page down. If we don't need the element anymore and we are not animating it disappear (e.g. with [a CSS `transition`](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions/Using_CSS_transitions)), it should be safe to remove it